src/Components/MessagePost.js

```diff
@@ -63,12 +67,19 @@ class MessagePost extends Component {
                             <p className="message-post-message">{this.props.content}</p>
                         </MDBCol>
                     </MDBRow>
+
+                    <MDBRow>
+                        <MDBCol size="12">
+                            <MDBIcon 
```

Comment:
Can we add some text in this row saying `"X replies"` to match figma designs? Hopefully doesn't add too much complexity since we can expose the size of the comments array as a state?

Reply:
If you want to do this as another refactor PR that is OK with me too. Just add the todo in the comments and add it to the google doc.